### DISCUSSION TITLE
I would like to submit a OSP model snapshot

### DISCUSSION THREAD

**Discussion by OliversLab** [Open-Systems-Pharmacology/Forum#899] [Models] [answered]
Dear all,

I would like to contribute and deposit the first version of our Cabozantinib PBPK model. Would this be possible?

name of your model: Cabozantinib

short description of the model repository:
# Cabozantinib PBPK

Within this repository we share whole body PBPK model of the TKI Cabozantinib with enterohepatic recirculation (EHC) and drug-drug-interaction (DDI) via CYP3A4 inducer. The model was developed using available pharmacokinetic studies. The model describes the EHC and the DDIs with Rifampin as CYP3A4 inducer. 

For a detailed documentation of model development, quantitative model evaluation and sensitivity analysis, please refer to [1].

## License

The model code is distributed under the [GPLv2 License]

## References

[[1] Gerner, B.; Scherf-Clavel, O. Physiologically Based Pharmacokinetic Modelling of Cabozantinib to Simulate Enterohepatic Recirculation, Drug-Drug Interaction with Rifampin and Liver Impairment. *Pharmaceutics* **2021**, *XX*, XXX.](https://www.mdpi.com/1999-4923/xxxx)

repository topics: pbpk, ddi

Thank you very much,
best regards,
Oliver

**Comment by Yuri05** [comment, marked as answer]
@OliversLab I have created new repository [Cabozantinib-Model](https://github.com/Open-Systems-Pharmacology/Cabozantinib-Model)
You can add your files as described in [CREATING_MODEL_REPOSITORY](https://github.com/Open-Systems-Pharmacology/Suite/blob/develop/CREATING_MODEL_REPOSITORY.md)
Thanks for your contribution.

**Comment by OliversLab** [comment]
Thank you so much.
Have a nice weekend.
Oliver
